frontend/gateway/container.go

```diff
@@ -227,7 +227,7 @@ func (gwCtr *gatewayContainer) Start(ctx context.Context, req client.StartReques
 	if procInfo.Meta.Cwd == "" {
 		procInfo.Meta.Cwd = "/"
 	}
-	procInfo.Meta.Env = addDefaultEnvvar(procInfo.Meta.Env, "PATH", utilsystem.DefaultPathEnv)
+	procInfo.Meta.Env = addDefaultEnvvar(procInfo.Meta.Env, "PATH", utilsystem.DefaultPathEnvUnix) // support windows?
```

Comment:
@coryb Discovered that there is no way to check platform here. Do you think it may become a problem in the future and should we extend this somehow?

Reply:
I think we can plumb the platform information through `NewContainer` if we need to.  It seems like something that would not be to hard to add on later?

Reply:
@coryb I think we should do it. It can be optional so doesn't complicate the client side. Another case where it may come up is if in the future there is a fully distributed mode and BuildKit needs to chose the correct node for the container (eg. windows/linux or arch based). so basically should add both of these fields https://github.com/moby/buildkit/blob/master/solver/pb/ops.proto#L21-L22

Reply:
Sure, I can file an issue and take that on after this PR gets merged.  Sound good?